src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,55 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs based on the following:
+  * http://stackoverflow.com/questions/2315912/scala-best-way-to-parse-command-line-parameters-cli
+  * @constructor Create a new command reader
+  */
+object CommandReader {
+  val usage =
+    """
+       Usage: <spark arguments> sparkRocks-assembly-1.0.jar -inputFile <inputFile> -numberSeedJoints <Int> <output opts>
+         where:
+              <spark arguments>    See www.spark.apache.org for details on how to submit Spark applications
+              <inputFile>          File that contains input information on the rock volume of interest and joint sets
+              <Int>                Integer that specifies the number of joints to process before initiating parallel
+                                   instances
+         and <output opts> is one or all of the following
+              -toVTK               Generate output that can be converted to VTK format by rockProcessor
+              -toIE                Generate output that uses inequalities to represent rock blocks along with the
+                                   blocks' center of mass
+    """.stripMargin
+
+  type OptionMap = Map[Symbol, String]
+
+  def parseArguments(inputs: Array[String]): OptionMap = {
+    if (inputs.length == 0) {
+      println(usage)
+      sys.exit(0)
+    }
+    val inputList = inputs.toList
+
+    def nextOption(inputMap: OptionMap, inputsString: List[String]): OptionMap = {
+      inputsString match {
```

Comment:
You know, the more I look at this the more confused I get. And I think it still won't work all the time. I could do `-inputFile someFile -toVTK` and it would match the third case, match the sixth case, and match the first case with no error occurring.

 I think we should just simplify it down so that each `case` is a valid sequence of arguments. Then we don't need this `nextOption` function and we don't need any recursion. So what I'm saying is we'd have something like
```scala
inputs.toList match {
  case "-inputFile" :: file :: "-numberSeedJoints" :: seeds :: "-toVTK" => ...
  case "-inputFile" :: file :: "-numberSeedJoints" :: seeds :: "-toIE" => ...
  case "-inputFile" :: file :: "-numberSeedJoints" :: seeds :: "-toVTK" :: "-toIE" => ...
```
This probably still could be improved, but let's not worry too much about it for now.

Also, what does `toIE` mean? Aren't we still doing JSON?